packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1374,6 +1408,27 @@ export function _useFakeChatAdapters(args: _FakeChatAdapterArgs): _FakeChatAdapt
 // @beta
 export const useTeamsCallAdapter: (args: Partial<TeamsCallAdapterArgs>, afterCreate?: ((adapter: TeamsCallAdapter) => Promise<TeamsCallAdapter>) | undefined, beforeDispose?: ((adapter: TeamsCallAdapter) => Promise<void>) | undefined) => TeamsCallAdapter | undefined;
 
+// @public
+export interface VideoBackgroundBlurEffect {
+    // (undocumented)
+    effectType: 'Blur';
+}
+
+// @beta
+export interface VideoBackgroundEffectsOptions {
+    backgroundImages?: BackgroundImage[];
+    backgroundUploadHandler?: BackgroundUploadHandler;
+    disableVideoEffects?: boolean;
+}
+
+// @public
+export interface VideoBackgroundReplacementEffect {
+    // (undocumented)
+    customBackground: CustomBackground;
```

Comment:
we should avoid the word `custom` here as calling never uses custom.
Instead just `background: VideoBackgroundImage`